Code golf, add new schemes, conformed old schemes, alphabetized lists…

TPD94 opened the pull request:
…, better errors.

Commit:
Code golf, add new schemes, conformed old schemes, alphabetized lists, better errors.

Commit:
Added sling

FoxRefire commented:
Thanks, LGTM.

> alphabetized lists

I intended to fix this later, but because `selectRules.conf` is evaluated in order from the top line, if we do this without changing the prioritization mechanism, more general rules will be evaluated first, causing problems with automatic selection that are not correct.
Please include a workaround for this, or leave it in its original state for now.

FoxRefire committed:
TODOFIX: Revert selectRules.conf to original order

Priority: P3
This should be neatly ordered in the future, but first we need to implement a proper ordering algorithm.

FoxRefire committed:
Vocablary

https://github.com/FoxRefire/wvg/commit/bbc4cb81bad71a1444a7c46c94cc6a36140a24c8

FoxRefire committed:
Delete .gitattributes
